src/main/java/core/service/impl/Strategy.java

```diff
@@ -0,0 +1,5 @@
+package core.service.impl;
+
+public interface Strategy {
```

Comment:
Strategy used not correct in this case, Strategy can take parameter Operation and return Handler